src/helpers/getCurrenciesFromDomain.ts

```diff
@@ -0,0 +1,38 @@
+import { Asset, StellarTomlResolver } from "stellar-sdk";
+
+export const getCurrenciesFromDomain = async (
+  homeDomain: string,
```

Comment:
it would actually be great to share this function in Freighter (and other apps that need it). I wonder if it makes sense to add this to stellar-sdk or some other shared util library. I'll create a ticket for myself to start thinking about that. 

I'm sure there are other instances where we're duplicating a lot of the same things

Reply:
💯 Agree with this.

Reply:
💯 It would be awesome to add this to the js-stellar-sdk.